widget/view.html

```diff
@@ -254,386 +235,400 @@ <h6 class="font-weight-normal margin-bottom-sm modal-title text-transform-none">
-                                        <uib-accordion-heading class="padding-md">
-                                            <div data-ng-click="toggleFeedRules(feedRules.highConfidenceThreatFeeds, $event)"
-                                                data-ng-class="{'queue-text-active-color': feedRules.highConfidenceThreatFeeds.open}">
-                                                <div class="margin-right-lg pull-left">
-                                                    <span class="queue-card-actions">&nbsp;<i
-                                                            class="fa {{ feedRules.highConfidenceThreatFeeds.open===true ? 'fa-chevron-down' : 'fa-chevron-right'}}"></i></span>
-                                                </div>
-                                                <div class=" pull-left queue-card-header-text margin-right-3">High
-                                                    Confidence Threat Feed Block</div>
-                                                <div class="pull-right margin-right-md">
-                                                    <label
-                                                        class="margin-0 switch switch-pill switch-label switch-success">
-                                                        <input type="checkbox" class="switch-input"
-                                                            id="high-confidence-threat-feeds-switch-slider"
-                                                            name="high-confidence-threat-feeds-switch-slider"
-                                                            data-ng-model="feedRules.highConfidenceThreatFeeds.enabled" />
-                                                        <span class="switch-slider" data-checked="Enabled"
-                                                            data-unchecked="Disabled"></span>
-                                                    </label>
-                                                </div>
-                                                <div class="clearfix"></div>
-                                            </div>
-                                        </uib-accordion-heading>
-                                        <p class="font-Size-13 muted margin-top-negative-20"><span>This rule will Block
-                                                Enriched Malicious Feeds with confidence greater than specified
-                                                threshold.</span></p>
-                                        <div class=" form-group margin-top-negative-10 padding-left-lg padding-right-lg overflow-auto query-section"
-                                            data-ng-class="{ 'has-error':
+                                </div>
+                            </uib-accordion-heading>
+                            <div>
+                                <p class="font-Size-13 muted margin-top-negative-20"><span>This rule will Block
```

Comment:
I can see messages and texts are part of this html only and hence it will not comply for i18n.
Have we added i18n support for this wizard?

Reply:
Wizard text are not finalized hence not added i18n yet

Reply:
please file a mantis to track this change